Makefile

```
@@ -740,3 +740,64 @@ test-mcp-vscode: start-service build-mcp ## Run VS Code/Cursor MCP test scripts
 	status=$$?; \
 	$(MAKE) stop-mcp stop-service; \
 	exit $$status
+
+## ------------------------------------------------------------------------------------------------
+## Atris Upgrade Tests
+## ------------------------------------------------------------------------------------------------
+
+.PHONY: run-pre-upgrade run-post-upgrade-verify run-post-upgrade run-post-upgrade-cleanup run-atris-upgrade
+
+run-pre-upgrade:
```

checkmake (0.3.2)

[warning] Target body for "run-pre-upgrade" exceeds allowed length of 5 lines (14).

(maxbodylength)